projects/opendr_ws/README.md

```diff
@@ -49,8 +49,8 @@ For the initial setup you can follow the instructions below:
    You can now run an OpenDR ROS node. More information below.
 
 #### After first time setup
-For running OpenDR nodes after you have completed the initial setup, you can skip step 0 from the list above.
-You can also skip building the workspace (step 4) granted it's been already built and no changes were made to the code inside the workspace, e.g. you modified the source code of a node.
+For running OpenDR nodes after you have completed the initial setup, you can skip steps 2 and 5 from the list above.
+You can also skip building the workspace (step 6) granted it's been already built and no changes were made to the code inside the workspace, e.g. you modified the source code of a node.
```

Comment:
This last commit [623c80a](https://github.com/opendr-eu/opendr/pull/359/commits/623c80ae46e67f4dd6bd7fb3ca42b6d560afa02f) seems to revert this section to an older version. Did you apply the fixes from an outdated branch maybe? Similarly, on the list below some node  entries are removed.

Reply:
I think I have reverted the file from develop branch. I will check it now again and fix the text. Interesting git stuff happening :D